analysers/analyser_merge_traffic_signs.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, config, error_file, logger, classs, level, otype, conflation,
 
         self.missing_official = self.def_class(item = 8300, id = classs, level = level, tags = ['merge', 'leisure'],
             title = T_f('Unmapped {0}', title),
-            detail = T_f('Traffic sign ({1}) detected by Mapillary, but no nearby tagging of any:{0}', '\n' + ',\n'.join(missing_tags), title),
+            detail = T_f('Traffic sign ({1}) detected by Mapillary, but no nearby tagging of any:{0}', '\n- ' + '\n- '.join(missing_tags), title),
```

Comment:
```suggestion
            detail = T_f('Traffic sign ({1}) detected by Mapillary, but no nearby tagging of any:{0}', '\n\n ' + '\n- '.join(missing_tags), title),
```

Reply:
This would remove the bullet from the first item, but will add an extra EOL in 👍 

```suggestion
            detail = T_f('Traffic sign ({1}) detected by Mapillary, but no nearby tagging of any:{0}', '\n\n- ' + '\n- '.join(missing_tags), title),
```